Documentation/devel/stage1-implementors-guide.md

```diff
@@ -63,18 +62,18 @@ rkt supports two ways of doing that, stage1 implementors must pick one:
 * `--interactive` to run a pod interactively, that is, pass standard input to the application (only for pods with one application)
 * `--local-config=$PATH` to override the local configuration directory
 
-### `rkt enter` => "coreos.com/rkt/stage1/enter"
+### `rkt enter` => `coreos.com/rkt/stage1/enter`
 
 0. rkt verifies the pod and image to enter are valid and running
-1. chdirs to "/var/lib/rkt/pods/$uuid"
-2. resolves the "coreos.com/rkt/stage1/enter" entrypoint via Annotations found within "/var/lib/rkt/pods/$uuid/stage1/manifest"
-3. executes the resolved entrypoint relative to "/var/lib/rkt/pods/$uuid/stage1/rootfs"
+1. chdirs to `/var/lib/rkt/pods/$uuid`
+2. resolves the `coreos.com/rkt/stage1/enter` entrypoint via annotations found within `/var/lib/rkt/pods/$uuid/stage1/manifest`
+3. executes the resolved entrypoint relative to `/var/lib/rkt/pods/$uuid/stage1/rootfs`
 
 In the bundled rkt stage 1, the entrypoint is a statically-linked C program found at `/enter` within the stage 1 ACI rootfs.
 This program enters the namespaces of the systemd-nspawn container's PID 1 before executing the `/appexec` program.
 `appexec` then `chroot`s into the ACI's rootfs, loading the application and its environment.
 
-An alternative stage 1 would need to do whatever is appropriate for entering the application's environment created by its own "coreos.com/rkt/stage1/run" entrypoint.
+An alternative stage 1 would need to do whatever is appropriate for entering the application's environment created by its own `coreos.com/rkt/stage1/run` entrypoint.
```

Comment:
1. "stage1" or "stage 1" (make consistent)
2. s/application's/application/ (singular makes the most sense in this construction)
3. Would an alternative stage1's entrypoint still be referred to as `coreos.com/rkt/stage1/run` ?